src/condor_utils/file_transfer.cpp

```diff
@@ -4373,6 +4394,29 @@ FileTransfer::computeFileList(
 	ExpandFileTransferList( FilesToSend, filelist, preserveRelativePaths );
 	// dPrintFileTransferList( D_ZKM, filelist, ">>> computeFileList(), after ExpandeFileTransferList():" );
 
+	if (inHandleCommands && m_has_protected_url) {
```

Comment:
```suggestion
	// Presently, `inHandleCommands` will only be set on the shadow.  The conditional
	// here is abstractly, "if this side is telling the other side which URLs to download";
	// if it is, this side can take care of managing transfer tokens without the other side
	// having to do anything.  See the design doc for HTCONDOR-1819.  This logic
	// currently works because this function is only called on the upload side.
	if (inHandleCommands && m_has_protected_url) {
```

Reply:
I am tweaking a couple of words in this suggested comment so I will add to commit with other changes